File: problemas/iniciante/1007/1007.go
package main

import (
    "fmt"
)

func main() {
    var A,B,C,D int

    fmt.Scanf("%d", &A)
    fmt.Scanf("%d", &B)
    fmt.Scanf("%d", &C)
    fmt.Scanf("%d", &D)
    
    fmt.Printf("DIFERENCA = %d\n", (A * B - C * D))
}

File: problemas/iniciante/1007/1007.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val A: Int = sc.next().toInt()
    val B: Int = sc.next().toInt()
    val C: Int = sc.next().toInt()
    val D: Int = sc.next().toInt()

    println("DIFERENCA = " + (A * B - C * D))
}
